### PR TITLE
Merge opentracing-3.0 into mpOpenTracing-3.0

### PR DESCRIPTION
**Pull request by tevans78**
#build

### CODE REVIEW

File: dev/com.ibm.websphere.appserver.features/visibility/public/mpOpenTracing-3.0/io.openliberty.mpOpenTracing-3.0.feature [hunk 1]
```diff
@@ -7,13 +7,32 @@ IBM-App-ForceRestart: install, \
...
+    com.ibm.ws.jaxrs.defaultexceptionmapper.jakarta, \
+    io.openliberty.opentracing.2.0.internal.jakarta, \
+    io.openliberty.opentracing.2.0.internal.cdi.jakarta, \
+    io.openliberty.io.opentracing.opentracing-util.0.33.0, \
+    io.openliberty.opentracing.2.0.thirdparty; location:="dev/api/third-party/,lib/"; mavenCoordinates="io.opentracing:opentracing-api:0.33.0", \
+    com.ibm.ws.microprofile.opentracing.jaeger, \
+    com.ibm.ws.microprofile.opentracing.jaeger.adapter, \
+    com.ibm.ws.microprofile.opentracing.jaeger.adapter.impl,\
+    io.openliberty.opentracing.3.0.internal.restfulws, \
+    io.openliberty.opentracing.2.0.spi; location:="dev/spi/ibm/,lib/"
```
Comment (Azquelt) [inline dev/com.ibm.websphere.appserver.features/visibility/public/mpOpenTracing-3.0/io.openliberty.mpOpenTracing-3.0.feature:32]: This was in `jars` before, now it's in `bundles`. What's the difference?

Comment (tevans78) [reply dev/com.ibm.websphere.appserver.features/visibility/public/mpOpenTracing-3.0/io.openliberty.mpOpenTracing-3.0.feature:32]: The jar in question was actually an OSGi bundle, that is why it moved. While we have observed that the spi package is also exported by `io.openliberty.opentracing.2.0.internal`, that should not be a problem for now. In the long run, the spi package should be exported solely by the spi bundle.

Comment (tevans78) [reply dev/com.ibm.websphere.appserver.features/visibility/public/mpOpenTracing-3.0/io.openliberty.mpOpenTracing-3.0.feature:32]: https://github.com/OpenLiberty/open-liberty/issues/19290

**Review by Azquelt — APPROVED** [review]

**Comment by tevans78** [comment]
#libby

**Pull request merged**